### PR TITLE
gibo: add page

### PR DESCRIPTION
**Pull request by rootbeersoup**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/gibo.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+`gibo list`
+
+- Write a boilerplate to stdout:
+
+`gibo dump {{boilerplate}}`
+
+- Write a boilerplate to .gitignore:
+
+`gibo dump {{boilerplate}} >> {{.gitignore}}`
```
Comment (sbrl) [inline pages/common/gibo.md]: On some Linux systems, it's an error to have a space after the `>>`.

Comment (rootbeersoup) [reply pages/common/gibo.md]: Ah, didn't know that. Would `gibo dump {{boilerplate}} >>{{{.gitignore}}` be correct?

Comment (sbrl) [reply pages/common/gibo.md]: If you remove the 3rd `{`, then yep :slightly_smiling_face:

Comment (rootbeersoup) [reply pages/common/gibo.md]: Oops. Didn't make it into the commit.

**Pull request merged**